Sources/PaymentButtons/PaymentButton+ImageAsset.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+extension PaymentButton {
+
+    // MARK: - Internal Properties
+    
+    var image: UIImage? {
+        UIImage(named: fileName, in: PaymentButton.bundle, compatibleWith: nil)
+    }
+    
+    var imageAccessibilityLabel: String {
+        // NEXT_MAJOR_VERSION: - To be replaced with translation strings.
+        if fileName.starts(with: "credit") {
+            return "PayPal Credit"
+        } else {
+            return "PayPal"
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    /// Name of the sized `.imageset` assets within `Assets.xcassets` directory
+    private var fileName: String {
+        var imageAssetString = ""
+        switch fundingSource {
+        case .payPal:
+            imageAssetString += "paypal_"
+
+            if size == .mini {
+                imageAssetString += "monogram_"
+            }
+
+        case .payLater:
+            imageAssetString += "paypal_monogram_"
+
+        case .credit:
+            imageAssetString += "credit_"
+        }
+
+        switch color {
+        case .gold, .white, .silver:
+            imageAssetString += "color"
+
+        case .black, .darkBlue:
+            imageAssetString += "monochrome"
+
+        case .blue:
+            imageAssetString += "blue"
+        }
+        
+        return imageAssetString
```

Comment:
Can 🏈 on this but is it worth doing a check for an empty `imageAssetString` anywhere? Is there ever an instance where we'd fall through all of these switches and end up with the empty string returned from line 24?

Reply:
Yeh, agreed. This empty string default isn't a good practice and not very error-proof. I know Stephanie's PR here https://github.com/paypal/paypal-ios/pull/235/files updated these to be full string names that match the asset files, versus concatenating them - which is an improvement.

But we should def further clean this up. Gonna leave it for a separate PR 🏈